pkg/api/convertertoapi_test.go

```diff
@@ -946,9 +950,9 @@ func TestConvertVLabsWindowsProfile(t *testing.T) {
 	}
 
 	for _, c := range cases {
+		c := c
```

Comment:
@marosset FYI. This must be set to capture range variable. Otherwise, only one case will be run multiple times.
Reference: https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks